Content.Client/CriminalRecords/CriminalRecordsConsoleWindow.xaml.cs

```diff
@@ -55,6 +62,8 @@ public CriminalRecordsConsoleWindow(EntityUid console, uint maxLength, IPlayerMa
         _proto = prototypeManager;
         _random = robustRandom;
         _accessReader = accessReader;
+        _entManager = IoCManager.Resolve<IEntityManager>();
+        _spriteSystem = _entManager.System<SpriteSystem>();
```

Comment:
Better to use injectdependencies or pass it in, never resolve.

Reply:
I did it for the ent but can’t do it for sprite system. Hope that’s ok, it’s the way all the other comps that use it do it from what I can see.